wiki/Configuration:-Layout.md

```diff
@@ -169,7 +169,7 @@ layout {
         inactive-color "#505050"
 
         // active-gradient from="#ffbb66" to="#ffc880" angle=45 relative-to="workspace-view"
-        // inactive-gradient from="#505050" to="#808080" angle=45 relative-to="workspace-view"
+        // inactive-gradient from="#505050" to="#808080" angle=45 relative-to="workspace-view" gradient-type="linear"
```

Comment:
I think it would make sense for this to instead be spelled like `in="srgb"`, `in="srgb-linear"`, `in="oklch shorter hue"`, mirroring [the CSS syntax](https://www.w3.org/TR/css-color-4/#color-interpolation-method).